rich/traceback.py

```diff
@@ -417,6 +426,20 @@ def safe_str(_object: Any) -> str:
                     msg=exc_value.msg,
                 )
 
+
+            if isinstance(exc_value, BaseExceptionGroup):
```

Comment:
It would be very easy to make this support older Python versions, with or without the `exceptiongroup` backport: 
```suggestion
            if sys.version_info[:2] < (3, 11):
                BaseExceptionGroup = getattr(sys.modules.get("exceptiongroup"), "BaseExceptionGroup", ())
            if isinstance(exc_value, BaseExceptionGroup):
```

Reply:
Yes, this was the plan but hasn't been implemented because I wanted to get a PoC out first with some feedback. See my first post as well at the bottom.